sys/txs-compiler/src/TorXakis/Parser/BExpDecl.hs

```diff
@@ -120,24 +120,35 @@ actOfferP = ActOfferDecl <$> offersP <*> actConstP
 
 -- | Parser for offers.
 offersP :: TxsParser [OfferDecl]
-offersP =  predefOffer "ISTEP"
-       <|> predefOffer "QSTEP"
-       <|> predefOffer "HIT"
-       <|> predefOffer "MISS"
-       <|> offersP'
+offersP = optional (txsSymbol "{")
```

Comment:
It seems { A? x | B ? y  is allowed.
I.e. non-matching brackets.

Reply:
Yes! Good catch. I corrected it.

BTW, it is outside of the scope of this issue, but why do we need curly braces at all?